streams/src/main/java/org/apache/kafka/streams/kstream/internals/StatelessProcessorNode.java

```diff
@@ -41,31 +40,28 @@
     private List<String> multipleParentNames = new ArrayList<>();
 
 
-    StatelessProcessorNode(final String parentProcessorNodeName,
-                           final String processorNodeName,
-                           final ProcessorSupplier<K, V> processorSupplier,
+    StatelessProcessorNode(final String nodeName,
+                           final ProcessorParameters processorParameters,
                            final boolean repartitionRequired) {
 
-        super(parentProcessorNodeName,
-              processorNodeName,
+        super(nodeName,
               repartitionRequired);
 
-        this.processorSupplier = processorSupplier;
+        this.processorParameters = processorParameters;
     }
 
-    StatelessProcessorNode(final String parentProcessorNodeName,
-                           final String processorNodeName,
+    StatelessProcessorNode(final String processorNodeName,
```

Comment:
I saw `processorNodeName` in this class but in some other classes they are `nodeName` are the the same? If yes please make sure naming terminologies are consistent.

Reply:
ack fixed